core/src/main/java/org/apache/accumulo/core/conf/Property.java

```diff
@@ -1315,6 +1345,8 @@ private void precomputeAnnotations() {
     ReplacedBy rb = getAnnotation(ReplacedBy.class);
     if (rb != null) {
       replacedBy = rb.property();
+    } else {
+      isReplaced = false;
```

Comment:
What prompted this change?

Reply:
Thanks for the explanation.  I did not understand the change and I never would have figured that out.